### ISSUE TITLE
[Bug]: Attu hangs when building an index for an INT64 field

### ISSUE THREAD

**Issue by changhong52** [milvus-io/milvus#21598]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:  2.2.0
- Deployment mode(standalone or cluster):  cluster
- MQ type(rocksmq, pulsar or kafka):    pulsar
- SDK version(e.g. pymilvus v2.0.0rc2):  V2.2.0
- OS(Ubuntu or CentOS):  Ubuntu
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

创建号collection后手动在ATTU给int64类型的标量创建索引，一直转圈圈

### Expected Behavior

期望给标量建索引可以很快

### Steps To Reproduce

```markdown
1、创建collection并插入数据
2、在ATTU里release数据
3、点击创建标量索引
4、一直转圈圈
```

### Anything else?

![image](https://user-images.githubusercontent.com/121591595/211308229-c144ef8a-427b-4262-b364-7580c363fc30.png)

**Comment by shanghaikid** [comment]
- What's your attu version? 
- How much data do you have in your collection?

**Comment by Cheung7788** [comment]
log addr：https://drive.google.com/drive/u/0/folders/1I8gJ3Q81yyqAAG7GIEg-0vi0nlH8kPh_

**Comment by shanghaikid** [comment]
> log addr：https://drive.google.com/drive/u/0/folders/1I8gJ3Q81yyqAAG7GIEg-0vi0nlH8kPh_

Sorry, we can access the file.

**Comment by Cheung7788** [comment]
please try new link

**Comment by changhong52** [comment]
> * 你的attu版本是多少？
> * 您的收藏中有多少数据？

![image](https://user-images.githubusercontent.com/121591595/211448814-a1a7076d-adf0-426a-8d7f-2674c71deb8e.png)

**Comment by changhong52** [comment]
> * 你的attu版本是多少？
> * 您的收藏中有多少数据？

attu版本：v2.2.1 数据50W

**Comment by shanghaikid** [comment]
I tried attu v2.2.2 and milvus v2.2.2 standalone, importing 550,200 entities,  it finished indexing for about 10s on my laptop.

![image](https://user-images.githubusercontent.com/185051/211449541-abcc0411-66ba-41b2-8e9b-baf2ba5b44a8.png)
![image](https://user-images.githubusercontent.com/185051/211449564-db8b98e2-7e20-4fde-9c16-5987fcb7088b.png)

@yanliang567 do we have some fix about indexing scalar field in v2.2.2?  or can you test the case on a cluster version ?
